packages/compass-crud/src/components/element.less

```diff
@@ -56,12 +56,6 @@
 
     &-is-objectid {
       color: orangered;
-      &::before {
-        content: 'ObjectId("';
-      }
-      &::after {
-        content: '")';
-      }
```

Comment:
Made sense to me to move this implementation to the component itself. Maybe later down the road we can do the same for some other bson types we render like UUID or Symbol